pom.xml

```diff
@@ -63,7 +63,7 @@
     </pluginRepositories>
     <properties>
         <changelist>999999-SNAPSHOT</changelist>
-        <jenkins.version>2.289.1</jenkins.version>
+        <jenkins.version>2.321</jenkins.version>
```

Comment:
I guess this should now go to 2.322 and the corresponding BOM.

Reply:
PR is up here: https://github.com/jenkinsci/workflow-support-plugin/pull/150